docker/hooks/build

```diff
@@ -1,9 +1,13 @@
 #!/bin/bash
 set -x
-# When we build 'latest' tag we want to not override BK_VERSION variable
+# When we build 'latest' tag we want always get the latest released version
+
 if [[ "$DOCKER_TAG" = "latest" ]]
 then
-  docker build -t $IMAGE_NAME .
+  # GH API currently returns tags in the order they would be returned by the "git tag" command, which means they are alphabetically sorted.
+  # BK_RELEASE_VERSION points to the "highest" released version
+  BK_RELEASE_VERSION=`curl -s https://api.github.com/repos/apache/bookkeeper/tags | jq -r '.[0].name' | sed 's/release-\([0-9.]*\).*/\1/'`
```

Comment:
Would it make more sense to rely on https://dist.apache.org/repos/dist/release/bookkeeper/,  https://downloads.apache.org/bookkeeper/, or https://archive.apache.org/dist/bookkeeper/? These urls are where the official releases are available for download.

Reply:
In the release guide, the tag is created in the last steps which means the releases are already available on the dist. So I think it is not a big deal?

Reply:
@michaeljmarshall Does that make sense to you?